docs/src/markdown-pages/aptitudes/window.listen-active-window.md

```diff
@@ -3,3 +3,5 @@ name: "Listen to Active Window changes"
 links_js: "listenActiveWindow"
 ---
 Receive notifications whenever the currently focused window changes.
+
```

Comment:
Would it be worthwhile to include this comment directly inside of the LDK code so typedoc can pull it in?